romancal/dq_init/tests/test_dq_init.py

```diff
@@ -155,7 +159,7 @@ def test_dq_add1_groupdq():
     dq[400, 500] = 3  # do_not_use and saturated pixel
 
     # write mask model
-    ref_data = maker_utils.mk_mask(shape=csize[1:])
+    ref_data = stnode.MaskRef.fake_data(shape=csize[1:])
```

Comment:
I think this is passing a node (not a datamodel) to a method. I suspect a datamodel might be the preferred structure here but to avoid unrelated changes this can be handled in a follow-up PR.